secure_message/resources/threads.py

```diff
@@ -118,7 +118,7 @@ class ThreadCounter(Resource):
     @staticmethod
     def get():
         surveys = request.args.getlist('survey')
-        is_closed = bool(strtobool(request.args.get('is_closed')))
+        is_closed = bool(strtobool(request.args.get('is_closed', False)))
```

Comment:
This generates a `AttributeError: 'bool' object has no attribute 'lower'` error, `False` needs to be a string.